DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -360,6 +355,92 @@ private int GetMetadataSheetId()
             return metadataSheet.Properties.SheetId.Value;
         }
 
+        private CellData MakeCellData(double number)
```

Comment:
Are you using `number`?

Reply:
I should be, thank you for spotting that! Fixed in c3e7c4b 🙂